lib/event_sourcery/event_processing/error_handlers/exponential_backoff_retry.rb

```diff
@@ -26,6 +26,7 @@ def with_error_handling
           report_error(error)
           update_retry_interval(error)
           sleep(@retry_interval)
+          EventSourcery.logger.info { "Retrying #{@processor_name} with error: #{error.message} at interval=#{@retry_interval}" }
```

Comment:
Perhaps it should log at `error` level instead of `info`?

Reply:
Its not really an error though, the error was the thing that caused the retry in the first place. This is more just info that indicates that its going to retry processing.